library/src/text/quotes.rs

```diff
@@ -139,6 +139,8 @@ impl<'s> Quotes<'s> {
         let region = region.as_ref().map(Region::as_str);
         let (single_open, single_close, double_open, double_close) = match lang.as_str() {
             "de" if matches!(region, Some("CH" | "LI")) => ("‹", "›", "«", "»"),
+            "es" if matches!(region, Some("MX")) => return Self::default(),
```

Comment:
We could drop support for es-MX since it uses the default quoting style.

Reply:
I think it is best to remove the check here, yes. You should only check for what is different from default, i.e, when it is Spain Spanish :)